.golangci-prod.toml

```diff
@@ -298,8 +298,6 @@
       "returnAfterHttpError",
       # Detects suspicious/confusing re-assignments.
       "sloppyReassign",
-      # Detects unsupported test and benchmark funcs.
-      "sloppyTestFuncName",
```

Comment:
context? why we removing this rule

Reply:
it's already deprecated by the newer version of golangci-lint